### PR TITLE
Adding Divi as a symlink example

### LINKED ISSUE #5413: Using Extensions That Assume Write Access Doc Update add divi
Re: [Using Extensions That Assume Write Access](https://pantheon.io/docs/assuming-write-access)

Priority: Low

## Issue Description:
Latest version of Divi 4.0.6 is gaining popularity in creating symlinks

## Suggested Resolution
Adding the symlinking procedures in doc for easy reference and guidance of CSE's

### PR DESCRIPTION
**Pull request by carl-alberto**
Closes #5413

## Effect
PR includes the following changes:
- Adding Divi as a symlink example

## Remaining Work
- [ ] List any outstanding work here
- [x] Technical review
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5414 | Difference |
| --- | --- | --- | --- |
| Performance Score | `68` | `49` | `-19` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5414` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17292/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17292/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5414 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.99999999999999` | `50` | `-7` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5414` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17295/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17295/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by alexfornuto** [comment]
Thanks @carl-alberto! In light of the ever-growing list of plugins/themes/modules that assume write access, I'm going to consolidate them all into one section of the Modules and Plugins with Known Issues doc, as we did for [Define FS_METHOD](https://pantheon.io/docs/modules-plugins-known-issues#define-fs_method).

As for this PR, I'm wary to have too many examples of the same solution (making a symlink), and would like to have a single general example that will serve the purpose. The example chosen should be both popular enough to be the plugin in question for a majority of readers, and generic enough that it will serve as an adequate example for others.

With that in mind I ask you: Is Divi that example? If not, which one is?

I welcome all input from @pantheon-systems/documentation and elsewhere.

**Comment by carl-alberto** [comment]
@alexfornuto yeah agreed not to put in many plugins with the similar solution, better yet if you agree  if we can show at least 4 or more variations that we mostly encounter(along with real-world examples for popular theme/plugins):

This is assuming that users are inside the `/code/` folder OR the base folder when they do a git clone of their repo locally from a Linux/Mac Terminal:

1) Symlinking Inside the wp-content folder

```
WP-Rocket:
ln -s ./uploads/cache ./wp-content/cache
ln -s ./uploads/wp-rocket-config ./wp-content/wp-rocket-config
```

2) Inside plugins folder

```
WooZone:
ln -s ./uploads/cache ./wp-content/plugins/woozone/cache
```

3) Inside themes

```
Uncode:
ln -s ../../../../uploads/uncode/assets/css ./wp-content/themes/uncode/core/assets
ln -s ../../../uploads/uncode/library/css ./wp-content/themes/uncode/library
```

4) From the wp root folder(same folder with the wp-config.php)

```
ln -s ./wp-content/uploads/needs_access ./needs_access
```

5) From a `nested docroot` setup where the `wp-content` folder is placed inside the `app` folder (usually nested docroot users knows already what they are doing, might be better to note it here to get them started)

```
ln -s ./uploads/needs_access ./web/app/needs_access

```


Would also need help in beta testing this plugin that I wrote that can further simplify the process of symlinking with WP, hopefully this will help out people that do not want to mess around with the command line https://wordpress.org/plugins/easy-symlinks and possibly get it documented if proven its worth

**Comment by alexfornuto** [comment]
Thanks for the additional clarity @carl-alberto! I can see why more examples is better for CSE. To resolve my concerns, I've (amongst other update) moved each example into an expandable panel.

If you're :+1: on my edits, I'll move this over for final review.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5414 | Difference |
| --- | --- | --- | --- |
| Performance Score | `81` | `57.99999999999999` | `-23` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5414` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17790/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17790/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by carl-alberto** [comment]
That new look is slick @alexfornuto +1 for me!

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5414 | Difference |
| --- | --- | --- | --- |
| Performance Score | `81` | `69` | `-12` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5414` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17883/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17883/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5414 | Difference |
| --- | --- | --- | --- |
| Performance Score | `82` | `77` | `-5` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5414` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17882/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17882/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/assuming-write-access.md [hunk 1]
```diff
@@ -4,81 +4,133 @@ description: Learn how to create symbolic links from the code directory to a fil
...
    cd wp-content/path/
    ```
 
 1. Create a symlink for the standard files path:
 
-    ```bash
+    ```bash{promptUser: user}
     # The first path will be used as the new file destination instead of whatever path the plugin assumed write access to
-    ln -s ../uploads/new-directory plugin-expects-to-write-to
+    ln -s ../uploads/new-directory #The last nested directory should mirror the directory name the plugin expects to write to
```
Comment (EdwardAngert) [inline source/content/assuming-write-access.md]: this example seems to be missing the link target

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5414 | Difference |
| --- | --- | --- | --- |
| Performance Score | `65` | `71` | `6` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `93` | `93` | `0` |

Lighthouse Audit Reports:

- [`pr-5414` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17890/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17890/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**